config/routes.rb

```diff
@@ -5,7 +5,9 @@
   ActiveAdmin.routes(self)
   devise_for :users
 
-  resources :articles
+  resources :articles do
+    resources :comments, module: :articles
```

Comment:
`only: [:create, :destroy]` ?

Reply:
What is the change here?

Reply:
check `rake routes` it must be creating routes for other methods like update, etc. that we don't have?